ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -32,14 +32,18 @@ http {
         ssl_prefer_server_ciphers on;
         proxy_ssl_session_reuse off;
 
-        # proxy to the web action path
         location / {
+            proxy_pass http://{{ groups['controllers']|first }}:{{ controller.port }};
+        }
+
+        # proxy to the web action path
+        location /web {
```

Comment:
@rabbah Does having this `/web` changes the vanity URL? and do we need to update the docs?

Today the docs say:
>https://guest.openwhisk.host/public/index.html becomes a alias for https://openwhisk.host/api/v1/experimental/web/guest/public/index.html
and 
also proxy https://guest.openwhisk.host to https://openwhisk.host/api/v1/experimental/web/guest/public/index.html

Does this now becomes ?
https://guest.openwhisk.host/public/index.html  -> https://guest.openwhisk.host/web/public/index.html 
and
https://guest.openwhisk.host -> https://guest.openwhisk.host/web